test/TimeoutTests/CSharp/HttpStart.cs

```diff
@@ -38,9 +35,9 @@ public static class HttpStart
 
         [FunctionName("StartAll")]
         public static async Task<IActionResult> StartAllTests(
-         [HttpTrigger(AuthorizationLevel.Function, methods: "get", Route = "start")] HttpRequest req,
-         [DurableClient] IDurableClient starter,
-         ILogger log)
+            [HttpTrigger(AuthorizationLevel.Function, methods: "get", Route = "start")] HttpRequest req,
+            [DurableClient] IDurableClient starter,
+            ILogger log)
```

Comment:
is this the result of a new auto-formatter? Otherwise, why did we add indentation here (and earlier in the file)?

Reply:
This was done by me because the existing indentation of a single space violated our normal C# conventions.